### PR TITLE
feature/XW-87/WCAG

### PR DESCRIPTION
**Pull request by remko48**
This pr contains fixes for the following errors:
image
image
image
image

### CODE REVIEW

File: pwa/src/templates/templateParts/header/HeaderTemplate.module.css [hunk 1]
```diff
@@ -40,3 +40,12 @@
...
 }
+
+.visuallyHidden {
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px, 1px, 1px, 1px);
+  color: red;
+}
```
Comment (lencodes) [inline pwa/src/templates/templateParts/header/HeaderTemplate.module.css:51]: Might be good to create a component for this purpose. Right now, when we want multiple hidden elements for navigational purposes, we would need to re-write this code.

Out of scope for this pull request, I think. But maybe nice for the backlog?

Comment (lencodes) [reply pwa/src/templates/templateParts/header/HeaderTemplate.module.css:51]: Do we need the `color` rule?

File: pwa/src/translations/en.ts [hunk 2]
```diff
@@ -32,4 +32,9 @@ export const en = {
   "No publication date available": "No publication date available",
   "No summary available": "No summary available",
   "Navbar-Logo": "Navbar-Logo",
+  "Opens a new window": "Opens a new window",
```
Comment (lencodes) [inline pwa/src/translations/en.ts:35]: "Opens _in_ a new window" might be better. If you disagree, feel free to not implement.

**Review by lencodes — APPROVED** [review]
Very nice! Learned a lot in this pull request 😄.

Approved, did leave some comments.

**Comment by remko48** [comment]
image
image
image
image

**Pull request merged**